packages/block-editor/src/components/block-breadcrumb/test/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * WordPress dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import BlockBreadcrumb from '../';
+
+describe( 'BlockBreadcrumb', () => {
+	test( 'should match snapshot', () => {
+		const wrapper = shallow( <BlockBreadcrumb /> );
+		expect( wrapper ).toMatchSnapshot();
```

Comment:
Personal preference:

Prefer using `react-testing-library` for testing. And for simple assertions like this, explicitly querying the root label and asserting it is more readable and simpler.

Reply:
Thanks @kevin940726 !

I, too, had this wish but I went for the lowest bar and copied from another dir. 😆 

I'll update.

Reply:
FYI, it's being [documented](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/testing-overview.md#javascript-testing) here. We still have tons of tests not refactored yet though 😅.